lxd/api_cluster.go

```diff
@@ -987,19 +987,17 @@ func tryClusterRebalance(d *Daemon) error {
 	leader, err := d.gateway.LeaderAddress()
 	if err != nil {
 		// This is not a fatal error, so let's just log it.
-		return errors.Wrap(err, "Failed to get current leader member")
+		return errors.Wrap(err, "failed to get current leader member")
```

Comment:
The convention in LXD is to use capital case for all error messages, even wrapped one. So this change should be reverted.